extensions/wikia/JSMessages/js/JSMessages.js

```diff
@@ -113,125 +92,26 @@
 		 * Load messages from given package(s) using content language
 		 */
 		JSMessages.getForContent = function(packages, callback) {
-			return JSMessages.get(packages, callback, window.wgContentLanguage);
+			return JSMessages.get(packages, callback, mw.config.get('wgContentLanguage'));
 		};
 
 		return JSMessages;
 	}
 
-	// private module for plural support
-	var Plurals = {
```

Comment:
It's provided by the core module `mediawiki.jqueryMsg`. Language data is loaded by ResourceLoader